src/components/AppHeader.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <header class="app-header">
+    <div role="banner" class="site-header">
+      <nav>
+        <router-link :to="{name: 'contact'}">Contact</router-link>
+      </nav>
+      <a href="/" class="logo"><img src="https://staticresources.rijkswaterstaat.nl/assets/img/logo.png?v=2.19.2" width="270" height="72" alt="Rijkswaterstaat Ministerie van Infrastructuur en Waterstaat." class="accessibility"></a>
+  </div>
+  <div id="menu" class="app-header__menu">
+    <div class="app-header__top-menu container">
```

Comment:
this is partially invisible on 320px

Reply:
<img width="327" alt="screen shot 2018-05-08 at 13 32 10" src="https://user-images.githubusercontent.com/4346054/39754990-5d2edcfa-52c4-11e8-9209-85e7415d9de9.png">